index.js

```diff
@@ -73,7 +74,8 @@ class S3Grc extends Base {
   }
 
   getDownloadUrl (filename, key, cb) {
-    const responseDisposition = `attachment; filename=${filename}`
+    const asciiFileName = filename.replace(/[\u{0080}-\u{FFFF}]/gu, '')
+    const responseDisposition = `attachment; filename=${asciiFileName}`
     const signedUrlExpireTime = 120
     const s3 = this.conf
```

Comment:
please write tests for it.

lets say you have a filename just containing non ascii chars… what happens then?

Reply:
ready, add a separate function as to make it more complete and tests for it.